.github/workflows/release-pypi.yml

```diff
@@ -81,64 +81,3 @@ jobs:
       with:
         user: __token__
         password: ${{ secrets.pypi_password }}
-
-  create-legacy-ckpt:
```

Comment:
AFAIK, this CI job has never run for 12+ months.

Reply:
what do you mean by not run, the last release was 1.4.x so a few months ago

Reply:
if you drop the automatic tool, then it will be you who will manually create a new checkpoint for each release?

Reply:
Will update!